Cockpit 224 (#335)
cockpit-project/cockpit-project.github.io: _posts/2020-07-22-cockpit-224.md

---

> title: Cockpit 224

Reviewer: How about devoting this blog post entirely to cockpit-podman 20, and drop the header prefixes? Or at least mention cockpit-podman in title:?

Reviewer 2: Yeah, makes sense!

author: kkoukiou
date: 2020-07-22
tags: cockpit linux
slug: cockpit-224
category: release
summary: Cockpit Podman new features and Cockpit bug fixes
comments: 'true'
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 224.

### Podman: Support checkpoint and restore operations for containers

> Checkpointing a container stops the container while writing the state of all processes in the container to disk.

Reviewer: ```suggestion
Checkpointing a container stops the container and writes the state of all container processes to disk.
```

Author: Decided to not include this commit in this release.

> With this a container can later be restored and continue running at exactly the same point in time as the checkpoint.

Reviewer: ```suggestion
Such a checkpoint can be restored later to resume the container in the exact state as it was at checkpoint time.
```

> This capability requires [CRIU 3.11](https://www.criu.org/Main_Page) or later installed on the system and currently

Reviewer: ```suggestion
This capability requires [CRIU 3.11](https://www.criu.org/Main_Page) or later installed. Currently
```

> only runc container runtime which is available on cgroupv1 systems is supported.

Reviewer: ```suggestion
only the `runc` container runtime is supported; this is only available with `cgroupv1` systems.
```

Thanks to Tomáš Glozar for this contribution!

### Podman: Show networking information for containers

Container details now contain basic networking information.

![Cockpit podman networking info](/images/podman-networking-info.png){:.screenshot}

Thanks to Eric Jeffrey for this contribution!

### Podman: Enable filtering images and containers by owner

> If user wants to work only with system or rootless containers now they can switch between these two modes.

Reviewer: ```suggestion
If user wants to work only with system or their own user containers, they can filter these now.
```

![Cockpit podman owner filter](/images/podman-owner-filter.png){:.screenshot}

> ### Podman: Don't show intermediate images by default

Reviewer: I'm fairly sure we never showed intermediate images.

```suggestion
### Podman: Optionally show intermediate images
```

> Intermediate images are not shown by default but are hidden and user can opt-in to see them.

Reviewer: ```suggestion
Intermediate images are hidden by default, but there is a new control to show them.
```

![Cockpit podman intermediate images](/images/podman-intermediate-images.png){:.screenshot}

### Podman : Enable setting up SELinux label when mounting volumes

> When creating new container and mounting volumes :z or :Z labels can be specified.

Reviewer: ```suggestion
When creating new container and mounting volumes, `:z` or `:Z` labels can be specified.
```
